test/views/html/employments/AddEmploymentStartDateFormViewSpec.scala

```diff
@@ -98,11 +98,10 @@ class AddEmploymentStartDateFormViewSpec extends TaiViewSpec {
       "there is a form with an error" in {
         def view: Html = template(formWithErrors, employmentName)
 
-        val errorMessage = doc(view).select(".error-message").text
-        val fieldSetError = doc(view).select("form > div").hasClass("form-group-error")
-
+        val errorMessage = doc(view).select(".govuk-error-message").text
+        val fieldSetError = doc(view).select("form div").hasClass("govuk-form-group--error")
         fieldSetError mustBe true
-        errorMessage mustBe globalErrorMessage
+        errorMessage contains globalErrorMessage
```

Comment:
how come this has changed?

Reply:
If I recall correctly it had something to do with an additional nested element appearing after implementing the new twirl component which caused the test to fail